src/main/resources/filter/metadata.filter

```diff
@@ -0,0 +1,9 @@
+attribute=machine.name
```

Comment:
Default is to filter all metadata? Maybe instead comment all line, but keep them so users know what metadata there is?